prow/plugins/plugins.go

```diff
@@ -385,6 +393,8 @@ type Blockade struct {
 	Explanation string `json:"explanation,omitempty"`
 }
 
+// Approve specifies a configuration for a single approve. The configuration for the
```

Comment:
In addition to keeping the first sentence on the top line. I would put subsequent sentences on the third+ lines (or at least second)